docs/_examples/evm_rpc_example.py

```diff
@@ -40,7 +40,10 @@ async def run(rpc_uri, quantity):
         ):
             logs.append(log)
 
+        block_from_timestamp = await rpc_client.get_block_by_timestamp(HexInt(1532118564))
```

Comment:
Please integrate the new method into the example, such as changing the example from a quantity of blocks to all block on or after a date/time.

Reply:
You added some code to get a block for a timestamp which didn't fit in with the overall example. Could you update the example from getting the logs for 100 blocks to getting the logs for a specified time range? The example will make more sense that way.

Reply:
Addressed.